index.html

```diff
@@ -0,0 +1,257 @@
+        <div class="professional_art_printing-section-items">
+            <div class="art_print_container">
+                <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                <p class="art_print_paragraph">
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="art_print_btn_container">
+                    <li><button class="transparent-btn">HTML</button></li>
+                    <li><button class="transparent-btn">Bootstrap</button></li>
+                    <li><button class="transparent-btn">Ruby</button></li>
+                </ul>
+            </div>
+            <div class="art-main-btn-container">
+                <button class="primarybtn">See Project</button>
+            </div>
+        </div>
+
+        <div class="professional_art_printing-section-items">
+            <div class="art_print_container">
+                <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                <p class="art_print_paragraph">
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="art_print_btn_container">
+                    <li><button class="transparent-btn">HTML</button></li>
+                    <li><button class="transparent-btn">Bootstrap</button></li>
+                    <li><button class="transparent-btn">Ruby</button></li>
+                </ul>
+            </div>
+            <div class="art-main-btn-container">
+                <button class="primarybtn">See Project</button>
+            </div>
+        </div>
+
+        <div class="professional_art_printing-section-items">
+            <div class="art_print_container">
+                <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                <p class="art_print_paragraph">
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="art_print_btn_container">
+                    <li><button class="transparent-btn">HTML</button></li>
+                    <li><button class="transparent-btn">Bootstrap</button></li>
+                    <li><button class="transparent-btn">Ruby</button></li>
+                </ul>
+            </div>
+            <div class="art-main-btn-container">
+                <button class="primarybtn">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <!-- DARK SECTION -->
+    <section class="dark_section">
+        <!-- ABOUT-ME SECTION -->
+        <section class="aboutme_section">
+            <div class="aboutme_container">
+                <h1 class="aboutme_header_text">About me</h1>
```

Comment:
- [ ] Kindly consider updating the about heading to match the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=yDKg5dMjRcmmZ2y5-0). Update the heading from `h1` to `h2` ``h3 etc... As each page should have only one `h1`. Also increase the font size and the font weigh to match the figma design. This will make your project look way more prettier
**Your project**
![image](https://user-images.githubusercontent.com/74411135/227732070-f879a59a-bfea-4fea-b117-675519ccbf65.png)

**The figma design**
![image](https://user-images.githubusercontent.com/74411135/227732058-5c4f045d-e82c-4d36-97b4-18b264aa1ec4.png)